seed/models/analysis_runs.py

```diff
@@ -0,0 +1,38 @@
+
+from django.contrib.postgres.fields import JSONField
+from django.db import models
+
+from seed.models import (
+    Analysis,
+    Cycle,
+    Property,
+    PropertyState
+)
+
+
+class AnalysisRun(models.Model):
+
+    CREATING = 10
+    READY = 20
+    QUEUED = 30
+    RUNNING = 40
+    FAILED = 50
+    STOPPED = 60
+    COMPLETED = 70
+
+    RUN_STATUSES = (
+        (CREATING, 'Creating'),
+        (READY, 'Ready'),
+        (QUEUED, 'Queued'),
+        (RUNNING, 'Running'),
+        (FAILED, 'Failed'),
+        (STOPPED, 'Stopped'),
+        (COMPLETED, 'Completed'),
+    )
+
+    analysis = models.ForeignKey(Analysis, on_delete=models.CASCADE)
+    cycle = models.ForeignKey(Cycle, on_delete=models.CASCADE)
+    property = models.ForeignKey(Property, on_delete=models.CASCADE)
+    property_state = models.ForeignKey(PropertyState, on_delete=models.CASCADE)
```

Comment:
When deciding to use CASCADE, I went with the approach of completing the following sentence: "If I delete the related [insert FK model], AnalysisRun should... ".

I ended up with "... also be deleted" myself, but I'm open to opposing thoughts.